src/features/SolarSystem/SolarRender.h

```diff
@@ -0,0 +1,345 @@
+#pragma once
+
+#include <Renderer.h>
+#include <gl\GL.h>
+#include <gl\GLU.h>
+#include"features\SolarSystem\SolarResource.h"
+namespace Features
+{
+	using namespace Interfaces;
+
+	class SolarSystemRenderer : public IRenderer
+	{
+	private:
+		float		g_Angle;
+		float 		g_Mercury;
+		float 		g_Venus;
+		float 		g_Earth;
+		float 		g_Mars;
+		float 		g_Jupiter;
+		float 		g_Saturn;
+		float 		g_Uranus;
+		float 		g_Neptune;
+		GLuint		gbTexture[10];
+		GLUquadric	*pQuadric[9];
+	public:
+		void update()
+		{
+			g_Angle = g_Angle + 0.1f;
```

Comment:
Can we move the implementation into its dedicated file before merging to master?

Reply:
It is done.